.travis.yml

```
@@ -0,0 +1,68 @@
+language: node_js
+os: linux
+sudo: false
+dist: trusty
+bundler_args: --retry 5
+node_js:
+  - "7"
+env:
+  global:
+    secure: dn0FPQ5IG4M/3kdwnyI78ElQ308Vc3QnKAvkWfwMFb8QxDqxQdnTo7AV1qTMtbLrDNkeEWIgi4nc7jmXNtvGTwOfhAULVh6606Qs5B+ezTdwzajbbFMI8SKQx/pnTojOMu8dx7V4lMoR/YWcojR0VC1IWVC62TGbSB1k5BDGgH0=
+git:
+  submodules: false
+before_install:
+  - sudo apt-get install -y xserver-xorg-dev libxext-dev libxi-dev
+install:
+  - npm install --silent -g phantomjs-prebuilt
+  - npm install --silent -g casperjs
+  - npm install --silent -g slimerjs
+  - npm install --silent -g gl
+edge: true
+notifications:
+  slack: metacell:5ALSeoP88DqIhORUJvxE56sq
+services:
+- docker
+addons:
+  firefox: "58.0"
+  apt:
+    packages:
+    - mesa-utils
+    - xvfb
+    - libgl1-mesa-dri
+    - libglapi-mesa
+    - libosmesa6
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - export SLIMERJSLAUNCHER=/home/travis/firefox-58.0/firefox/firefox
+  - export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu/
+
+script:
+  - travis_wait travis_retry docker build -t="hnn-ui" github.com/MetaCell/HNN-UI#skeleton -f="./Dockerfile_dev"
```

@afonsobspinto change skeleton to development after https://github.com/MetaCell/HNN-UI/pull/11
```suggestion
  - travis_wait travis_retry docker build -t="hnn-ui" github.com/MetaCell/HNN-UI#development -f="./Dockerfile_dev"
```